trunner/harness/plo.py

```diff
@@ -77,6 +78,7 @@ def enter_bootloader(self):
         except (pexpect.TIMEOUT, pexpect.EOF) as e:
             raise PloError("Failed to go into bootloader mode", output=self.dut.before) from e
 
+        time.sleep(0.1)
```

Comment:
Wouldn't it be better to use dut to expect a message that should be followed by a `\n`, instead of using a sleep, so we are likely covered for most cases, regardless of the host's speed?
For example, maybe `waiting for input `if I remember correctly.